functions/deleteTaskAPI.js

```diff
@@ -72,7 +71,75 @@ exports.deleteTask = functions.https.onRequest((request, response) => {
         });
         Activity.addActivity("DELETED", "Deleted task " + taskId, taskId, date, time);
 
-        const deleteTaskPromises = [p1, p2];
+        const p3 = db.collection("Main").doc("Trash").get().then((doc) => {
+            if (doc.exists) {
+                totalNumberOfTask = doc.data().TotalNumberOfTask;
+                totalDevelopmentTask = doc.data().TotalDevelopmentTask;
+                totalBusinessTask = doc.data().TotalBusinessTask;
+                totalMarketingTask = doc.data().TotalMarketingTask;
+                totalOtherTask = doc.data().TotalOtherTask;
+                totalCompletedTask = doc.data().TotalCompletedTask;
+                totalUnCompletedTask = doc.data().TotalUnCompletedTask;
```

Comment:
we dont need these counter totalCompletedTask, totalUnCompletedTask
and also change others name to totalDevelopmentTaskDeleted....